concepticondata/conceptlists/LSDO-2019-462.tsv

```diff
@@ -0,0 +1,463 @@
+ID	NUMBER	ENGLISH	AlternativeName	CONCEPTICON_ID	CONCEPTICON_GLOSS
+LSDO-2019-462-1	1	sky	sky	1732	SKY
+LSDO-2019-462-2	2	sun	sun	1343	SUN
+LSDO-2019-462-3	3	moon	moon	1313	MOON
+LSDO-2019-462-4	4	star	star	1430	STAR
+LSDO-2019-462-5	5	cloud	cloud (rain)	1489	CLOUD
+LSDO-2019-462-6	6	mist, fog	mist/fog	249	FOG
+LSDO-2019-462-7	7	rain	rain	658	RAIN (PRECIPITATION)
+LSDO-2019-462-8	8	rainbow	rainbow	1733	RAINBOW
+LSDO-2019-462-9	9	lightning	lightning (flashing)	1654	LIGHTNING
+LSDO-2019-462-10	10	thunder	thunder	1150	THUNDER
+LSDO-2019-462-11	11	shadow	shadow/shade	1388	SHADE
+LSDO-2019-462-12	12	wind	wind	960	WIND
+LSDO-2019-462-13	13	night	night	1233	NIGHT
+LSDO-2019-462-14	14	day	day	1225	DAY (NOT NIGHT)
+LSDO-2019-462-15	15	morning	morning	1339	MORNING
+LSDO-2019-462-16	16	noon	noon	12	MIDDAY
+LSDO-2019-462-17	17	yesterday	yesterday	1174	YESTERDAY
+LSDO-2019-462-18	18	tomorrow	tomorrow	1329	TOMORROW
+LSDO-2019-462-19	19	year	year	1226	YEAR
+LSDO-2019-462-20	20	east	east	1877	EAST
+LSDO-2019-462-21	21	west	west	1817	WEST
+LSDO-2019-462-22	22	water	water	948	WATER
+LSDO-2019-462-23	23	to be hot (water)	to be hot (water)	1286	HOT
+LSDO-2019-462-24	24	to be hot (person)	to be hot (person)	1286	HOT
+LSDO-2019-462-25	25	to be warm(water)	to be warm (water)	1232	WARM
+LSDO-2019-462-26	26	to be cold (person)	to be cold (person)	1287	COLD
```

Comment:
HOT and COLD here should only be linked to one item, I'd say. I'd even suggest, since they make the distinction, to add new concepts HOT (FEELING HOT) and COLD (FEELING COLD). Or do we have something similar already?

Reply:
I've added new concepts for this. @AnnikaTjuka, could you please double check? Thank you!